github/github.go

```diff
@@ -28,9 +28,10 @@ import (
 )
 
 const (
-	defaultBaseURL = "https://api.github.com/"
-	uploadBaseURL  = "https://uploads.github.com/"
-	userAgent      = "go-github"
+	CurrentVersion   = "45.2.0"
```

Comment:
Any ideas of where I can document this to make sure it gets updated on release? I guess the "Maintainer's Guide" in `CONTRIBUTING.md` could be a good place.

Reply:
How about we just change this to "Version" since that is pretty common?

Reply:
Oh, and sure... feel free to add a comment to CONTRIBUTING.md, although I don't think we have any written-up instructions for maintainers yet.  Maybe I should do that sometime.  :smile:

Reply:
Feel free to add a blank line after `Version` to uncouple it from the following unexported consts.

Reply:
Done! 🏁